### PR TITLE
Kotlin formatting

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1388)

### CODE REVIEW

File: analyzer/src/funTest/kotlin/integration/DrupalIntegrationTest.kt [hunk 1]
```diff
@@ -32,70 +32,72 @@ import java.io.File
...
         )
     }
 
     override val managedFilesForTest by lazy {
-        mapOf(PhpComposer.Factory() as PackageManagerFactory to
-                // Limit to definition files that come long with a lock file.
-                listOf(File(downloadResult.downloadDirectory, "composer.json")))
+        mapOf(
+            PhpComposer.Factory() as PackageManagerFactory to
+                    // Limit to definition files that come long with a lock file.
```
Comment (fviernau) [inline analyzer/src/funTest/kotlin/integration/DrupalIntegrationTest.kt]: This indentation is correct?

File: analyzer/src/main/kotlin/managers/GoDep.kt [hunk 2]
```diff
@@ -163,9 +163,12 @@ class GoDep(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: Rep
...
     private fun importLegacyManifest(lockfileName: String, workingDir: File, gopath: File) {
         if (lockfileName.isNotEmpty() && !File(workingDir, lockfileName).isFile &&
-                !analyzerConfig.allowDynamicVersions) {
-            throw IllegalArgumentException("No lockfile found in ${workingDir.invariantSeparatorsPath}, dependency " +
-                    "versions are unstable.")
+            !analyzerConfig.allowDynamicVersions
+        ) {
+            throw IllegalArgumentException(
+                "No lockfile found in ${workingDir.invariantSeparatorsPath}, dependency " +
+                        "versions are unstable."
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/GoDep.kt]: indent ok?

File: analyzer/src/main/kotlin/managers/NPM.kt [hunk 3]
```diff
@@ -129,12 +129,12 @@ open class NPM(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig:
...
 
     override fun mapDefinitionFiles(definitionFiles: List<File>) =
-            PackageJsonUtils.mapDefinitionFilesForNpm(definitionFiles).toList()
+        PackageJsonUtils.mapDefinitionFilesForNpm(definitionFiles).toList()
 
     override fun prepareResolution(definitionFiles: List<File>) =
-            // We do not actually depend on any features specific to an NPM version, but we still want to stick to a
-            // fixed minor version to be sure to get consistent results.
-            checkVersion(ignoreActualVersion = analyzerConfig.ignoreToolVersions)
+    // We do not actually depend on any features specific to an NPM version, but we still want to stick to a
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/NPM.kt]: just marking another type of indent which seems a bit surprising, haven't read the official style  guide ...

Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/NPM.kt]: Me neither 😀 I just had the IDE reformat the code as described at https://kotlinlang.org/docs/reference/code-style-migration-guide.html#migration-to-a-new-code-style.

File: analyzer/src/funTest/kotlin/integration/DrupalIntegrationTest.kt (hunk 1 above)
Comment (sschuberth) [reply analyzer/src/funTest/kotlin/integration/DrupalIntegrationTest.kt]: I did rely on the IDE's auto-formatting, see my comment below.

File: analyzer/src/main/kotlin/managers/GoDep.kt (hunk 2 above)
Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/GoDep.kt]: I did rely on the IDE's auto-formatting, see my comment below.

**Pull request merged**